core/src/main/scala/org/bitcoins/core/protocol/dlc/DLCPayoutCurve.scala

```diff
@@ -162,7 +162,7 @@ object OutcomePayoutMidpoint {
 }
 
 /** A single piece of a larger piecewise function defined between left and right endpoints */
-sealed trait DLCPayoutCurveComponent {
+sealed trait DLCPayoutCurveInterval {
```

Comment:
upon looking at this again, I don't personally find this name very descriptive. Sure there is an interval which corresponds to each of these but not every interval does, this very specifically corresponds to a single "piece" of the `DLCPayoutCurve` which is a piece-wise polynomial. Thoughts on `DLCPayoutCurvePiece`?